source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/stereolabs.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Configuration for Sterolabs Depth Cameras."""
+
+
+from omni.isaac.lab.sensors import CameraCfg, RayCasterCameraCfg, patterns
+from omni.isaac.lab.sim.spawners import PinholeCameraCfg
+
+##
+# Configuration as RayCasterCameraCfg
+##
+
+ZED_X_NARROW = RayCasterCameraCfg(
```

Comment:
Can we rename these to be the type of camera implementation as well? Something like `ZED_X_NARROW_RAYCASTER_CFG`? Also we end all configs with `CFG`.